src/Akka.Logger.NLog/Akka.Logger.NLog.csproj

```diff
@@ -7,6 +7,7 @@
     <Copyright>Copyright © 2013-2017 >Akka.NET Contrib Team</Copyright>
     <Authors>Akka.NET Contrib</Authors>
     <VersionPrefix>1.3.0</VersionPrefix>
+    <VersionSuffix>beta</VersionSuffix>
```

Comment:
NLog 5.0 with .NET Standard support is in a beta stage, so we should add a beta flag too